test/src/test/java/hudson/cli/CLIActionTest.java

```diff
@@ -122,16 +117,6 @@ public void serveCliActionToAnonymousUserWithoutPermissions() throws Exception {
         JenkinsRule.WebClient wc = j.createWebClient();
         // The behavior changed due to SECURITY-192. index page is no longer accessible to anonymous
         wc.assertFails("cli", HttpURLConnection.HTTP_FORBIDDEN);
-        // so we check the access by emulating the CLI connection post request
```

Comment:
Actually would pass again given the restored crumb exclusion, but is anyway redundant given more comprehensive, and realistic, tests later added to the CLI system.